### PR TITLE
[REF] .travis.yml: Using new template from MQT

### PR DESCRIPTION
**Pull request by moylop260**
* Exclude from global env instead of 'export'
 * Remove python-* apt packages to install them from pip with cache
 * Use python higher version
 * Fix pip cache
 * Install wkhtmltopdf from MQT instead of unpatched native version
 * Patch test_crawl in order to increase timeout
   * Fix https://github.com/OCA/OCB/issues/667
   * More details https://github.com/OCA/OCB/pull/744

### CODE REVIEW

File: .travis.yml
Comment (pedrobaeza) [inline .travis.yml]: Don't we use 0.12.1?

**Review by MiquelRForgeFlow — APPROVED** [review]

File: .travis.yml
Comment (moylop260) [reply .travis.yml]: I used `0.12.1` version but show the following error https://travis-ci.org/OCA/OCB/jobs/377321267#L664
and the other `.travis.yml` are using 0.12.4
 - [11.0 - 0.12.4](https://github.com/OCA/OCB/blob/dcdddbda379c9b14e66f0a8fd6d09820a7607306/.travis.yml#L20)
 - [10.0 - 0.12.4](https://github.com/OCA/OCB/blob/db85fb4f9429684bcf75f7cafb291f09fbd39888/.travis.yml#L22)

Comment (pedrobaeza) [reply .travis.yml]: OK, but if this is used on runbot, then there will be a difference on printed reports with Odoo runbot.

Comment (moylop260) [reply .travis.yml]: Yeah, I'm agree with you.
FYI this was disccussed from https://github.com/OCA/maintainer-quality-tools/pull/481#issuecomment-327259328

Review the `bad security issues` from @lasley comments
And issues related using old version from @Yajo comments

And the PR to support 0.12.1 version was refused.

Comment (pedrobaeza) [reply .travis.yml]: Well then, let's go for this...

**Review by pedrobaeza — APPROVED** [review]

**Comment by moylop260** [comment]
Wait not merge yet...
The modules are not installed :(

I'm looking for the issue here...

**Review by StefanRijnhart — APPROVED** [review]
Thanks!

**Pull request merged**
